README.md

```diff
@@ -2,6 +2,20 @@
 
 Launches "temporary" Jupyter notebook servers.
 
+**WARNING: tmpnb is no longer actively maintained.**
+
+We recommend you switch to using JupyterHub.
+
+### Configuration option 1:
+
+* JupyterHub with [tmpauthenticator](https://github.com/jupyterhub/tmpauthenticator)
```

Comment:
Let's go ahead and add a link to JupyterHub as well.

Reply:
Done!